cognee/modules/retrieval/exceptions/__init__.py

```
@@ -4,4 +4,4 @@
 This module defines a set of exceptions for handling various data errors
 """
 
-from .exceptions import SearchTypeNotSupported, CypherSearchError, CollectionDistancesNotFoundError
+from .exceptions import SearchTypeNotSupported, CypherSearchError
```

Ruff (0.11.9)

`.exceptions.SearchTypeNotSupported` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)